src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetFigureArea {
+    private double radius;
+
+    public void draw() {
+        System.out.println("Figure - circle: radius " + this.radius
+                + " ,area " + this.getArea() + " color "
+                + this.getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+    public Circle(double radius) {

```

Comment:
The constructor for `Circle` should also initialize the color if it is inherited from the `Figure` class. Consider adding a parameter for color and setting it using a setter method or directly if accessible.